tests/test_metrics.py

```diff
@@ -199,7 +199,7 @@ def test_pr(obs_series, mod_series):
     obs = obs_series
     mod = mod_series
 
-    pr = mtr.pr(obs, mod)
+    pr = mtr.pr(obs, mod, time=obs.index.to_series())
```

Comment:
seems strange - obs already have the time 🤔

Reply:
Sure, if the data is timeseries. Add some concurrent profile observations and consider once more if it is so strange.

Reply:
Last commit reverts to using the index.